flang/test/Lower/Intrinsics/execute_command_line.f90

```diff
@@ -39,7 +46,7 @@ end subroutine all_args
 subroutine only_command_default_wait_true(command)
 CHARACTER(30) :: command
 call execute_command_line(command)
-! CHECK-NEXT:     %c41_i32 = arith.constant 41 : i32 
+! CHECK-NEXT:     %c48_i32 = arith.constant 48 : i32 
```

Comment:
Using the name of the constant might be prone to error. Capture this in a value and use that value for checking.

Reply:
done